app/src/org/commcare/android/logic/GlobalConstants.java

```diff
@@ -50,4 +52,8 @@
     //All of the app state is contained in these values
     public static final String STATE_USER_KEY = "COMMCARE_USER";
     public static final String STATE_USER_LOGIN = "USER_LOGIN";
+
+    public static final String TRUSTED_SOURCE_PRIVATE_KEY = BuildConfig.TRUSTED_SOURCE_PRIVATE_KEY;
```

Comment:
we should rename this to `TRUSTED_SOURCE_PUBLIC_KEY` (since it's hq's public key we're defining here) to avoid confusion